templates/data/summary.html

```diff
@@ -24,25 +31,38 @@
           <tr>
             <td>artist</td>
             <td>
-              {% if lowlevel.metadata.tags.musicbrainz_artistid %}
-                <a href="https://musicbrainz.org/artist/{{ lowlevel.metadata.tags.musicbrainz_artistid[0] }}">
-                  {{ lowlevel.metadata.tags.artist[0] }}
-                </a>
-              {% else %}
-                {{ lowlevel.metadata.tags.artist[0] }}
-              {% endif %}
+              {%- if metadata -%}
+                {# TODO(roman): Handle multiple artists properly. #}
+                {%- set artist_id = metadata['artist-credit'][0]['artist']['id'] -%}
```

Comment:
I'd prefer to have this object navigation in the view instead of the template (or even better, a separate helper function)